src/forges/gitea.ts

```diff
@@ -37,6 +37,13 @@ export class GiteaForge extends Forge {
     sourceBranch: string;
     targetBranch: string;
   }): Promise<{ pullRequestLink: string }> {
+    await this.handleApiErrors(
```

Comment:
We already push changes (and thereby create a new branch with actual content) a few lines before, so this seems to be rather a hack that would not include those changes. Is gitea taking some time after pushing a branch to recognize it?

Reply:
These branches never get created though. Even if there would be a timely delay, the branches would exist in the forge then. But they are missing, indicating they never got created.

The build continues then and fails later when the branch is missing, so maybe this explicit creation is still solving the issue then?